images_config/kickstart/kickstart-robot.sh

```diff
@@ -0,0 +1,465 @@
+    dpkg -i /root/nomachine_${NOMACHINE_VERSION}_amd64.deb
+}
+
+function InstallNetData {
+    printHeader "InstallNetData"
+    apt-get install zlib1g-dev uuid-dev libmnl-dev gcc make git autoconf autoconf-archive autogen automake pkg-config curl -y
+    git clone https://github.com/firehol/netdata.git --depth=1 /root/netdata
+    cd /root/netdata
+    yes | ./netdata-installer.sh
+    cd
+    rm -r /root/netdata
+}
+
+function InstallCobCommand {
+    printHeader "InstallCobCommand"
+    if [ "$INSTALL_TYPE" == "master" ]; then
+        wget -O /usr/sbin/cob-command https://raw.githubusercontent.com/ipa320/setup_cob4/master/scripts/cob-command
+        chmod +x /usr/sbin/cob-command
+        sh -c 'echo "%users ALL=NOPASSWD:/usr/sbin/cob-command"' | sed -i -e "\|%users ALL=NOPASSWD:/usr/sbin/cob-command|h; \${x;s|%users ALL=NOPASSWD:/usr/sbin/cob-command||;{g;t};a\\" -e "%users ALL=NOPASSWD:/usr/sbin/cob-command" -e "}" /etc/sudoers
+    fi
+}
+
+function RemoveModemanager {
+    printHeader "RemoveModemanager"
+    apt-get purge modemmanager -y
+}
+
+function DisableUpdatePopup {
+    printHeader "DisableUpdatePopup"
+    sed -i 's/Prompt\=lts/Prompt\=never/g' /etc/update-manager/release-upgrades
+}
+
+# TODO: fxm ;)
+# Not functional under xenial
+function DisableFailsafeBoot {
+    printHeader "DisableFailsafeBoot"
+    sed -i 's/start on \(filesystem and static-network-up\) or failsafe-boot/start on filesystem and static-network-up/g' /etc/init/rc-sysinit.conf
+}
+
+function InstallAptCacher {
+    printHeader "InstallAptCacher"
+    #disable local forward to 10.0.1.1 cacher
+    if grep -q 'Acquire::http { Proxy "http://10.0.1.1:3142"; };' /etc/apt/apt.conf.d/01proxy ; then
+        rm /etc/apt/apt.conf.d/01proxy
+    fi
+    if grep -q 'Acquire::http::Proxy "http://10.0.1.1:3142/";' /etc/apt/apt.conf ; then
+        sed -i 's!Acquire::http::Proxy "http://10.0.1.1:3142/";!!g' /etc/apt/apt.conf
+    fi
+
+    HOSTNAME=$(cat /etc/hostname)
+    SERVERNAME=$(echo ${HOSTNAME%-*}-b1)
+    if [ "$INSTALL_TYPE" == "master" ]; then
+        apt-get install apt-cacher-ng -y
+        sed -i 's/\# PassThroughPattern: .\*/PassThroughPattern: .\*/g' /etc/apt-cacher-ng/acng.conf
+        systemctl restart apt-cacher-ng
+    elif [ "$INSTALL_TYPE" == "slave" ]; then
+        touch /etc/apt/apt.conf.d/01proxy
+
+        if grep -q 'Acquire::http { Proxy "http://10.0.1.1:3142"; };' /etc/apt/apt.conf.d/01proxy ; then
+            rm /etc/apt/apt.conf.d/01proxy
+            touch /etc/apt/apt.conf.d/01proxy
+        fi
+        if grep -q 'Acquire::http { Proxy "http://'$SERVERNAME':3142"; };' /etc/apt/apt.conf.d/01proxy ; then
+            echo "Proxy already in /etc/apt/apt.conf.d/01proxy, skipping InstallAptCacher"
+        else
+            echo 'Acquire::http { Proxy "http://'$SERVERNAME':3142"; };' >>  /etc/apt/apt.conf.d/01proxy
+        fi
+    fi
+}
+
+function InstallRealsense {
```

Comment:
should we put this function together with https://github.com/ipa320/setup_cob4/pull/199/files#diff-49b47db197119e8cdf6e298fb8c6df5eR34?